gradle.properties

```diff
@@ -17,3 +17,5 @@ POM_DEVELOPER_ID=xing-android
 POM_DEVELOPER_NAME=XING
 
 org.gradle.jvmargs=-Xmx2048M
+
+android.useAndroidX=true
```

Comment:
then you have to migrate from supportLib to AndroidX. https://developer.android.com/jetpack/androidx/migrate

Reply:
Yep, I was just making sure that the error I had in the CI was fixed.